packages/color-scale-block/package.json

```diff
@@ -0,0 +1,42 @@
+{
+    "name": "color-scale-block",
+    "private": true,
+    "author": "Frontify Developers <developers@frontify.com>",
+    "version": "1.0.0",
+    "main": "dist/index.js",
+    "scripts": {
+        "serve": "frontify-cli block serve",
+        "deploy": "frontify-cli block deploy",
+        "lint": "eslint src",
+        "lint:fix": "eslint --fix src",
+        "prettier": "prettier --check src",
+        "prettier:fix": "prettier --write src",
+        "typecheck": "tsc --noEmit"
+    },
+    "devDependencies": {
+        "@babel/core": "7.18.10",
+        "@cypress/react": "5.12.5",
+        "@frontify/eslint-config-react": "0.15.2",
+        "@frontify/frontify-cli": "5.1.1",
+        "@types/react": "17.0.47",
+        "autoprefixer": "10.4.8",
+        "cypress": "9.7.0",
+        "eslint": "8.21.0",
+        "postcss": "8.4.14",
+        "prettier": "2.7.1",
+        "tailwindcss": "3.1.7",
+        "typescript": "4.7.4"
+    },
+    "dependencies": {
+        "@frontify/app-bridge": "3.0.0-beta.10",
+        "@frontify/fondue": "9.11.0",
+        "@frontify/fondue-tokens": "3.0.1",
+        "@frontify/guideline-blocks-settings": "workspace:*",
+        "@frontify/guideline-blocks-shared": "workspace:*",
+        "react": "17.0.2",
+        "react-dnd": "^16.0.1",
+        "react-dnd-html5-backend": "^16.0.1",
+        "react-dom": "17.0.2",
+        "uuidv4": "^6.2.13"
```

Comment:
Can you create a utility instead, there is plenty of way to do it without using a library